tests/utils/test_StackArray.cpp

```diff
@@ -663,4 +663,226 @@ UTEST(test_MSArray, IterateWithFastIteratorWithGaps)
     {
         ASSERT_EQ(expected[i], actualResults[i]);
     }
+}
```

Comment:
the file naming between this and `SArray` tests could be clearer

Reply:
Renamed to `test_SArray` and `test_MSArray`